bridge/scripts/mint-script.ts

```diff
@@ -0,0 +1,286 @@
+        secretAccessKey: process.env.SAFE_OWNER_2_AWS_SECRET_ACCESS_KEY,
+        region: process.env.SAFE_OWNER_2_AWS_REGION!,
+        keyId: process.env.SAFE_OWNER_2_AWS_KEY_ID!,
+    },
+    provider
+);
+const owner3Signer = new AwsKmsSigner(
+    {
+        accessKeyId: process.env.SAFE_OWNER_3_AWS_ACCESS_KEY_ID,
+        secretAccessKey: process.env.SAFE_OWNER_3_AWS_SECRET_ACCESS_KEY,
+        region: process.env.SAFE_OWNER_3_AWS_REGION!,
+        keyId: process.env.SAFE_OWNER_3_AWS_KEY_ID!,
+    },
+    provider
+);
+
+const ethAdapterOwner1 = new EthersAdapter({
+    ethers,
+    signerOrProvider: owner1Signer,
+});
+
+const txServiceUrl = process.env.SAFE_TX_SERVICE_URL!;
+const safeService = new SafeServiceClient({
+    txServiceUrl,
+    ethAdapter: ethAdapterOwner1,
+});
+let safeSdkOwner1: Safe;
+let safeAddress: string;
+
+// If you have an existing Safe, you can use it instead of deploying a new one
+// https://app.safe.global/home?safe=eth:0x4A2FbE06004e37dE6Fe7Da59a53D14a407Def0ed
+const EXISTING_SAFE_ADDRESS = ethers.utils.getAddress(
+    process.env.SAFE_ADDRESS!
+);
+
+async function mint() {
+    const program = new Command();
+    program
+        .name("ad-hoc mint script")
+        .description("Transfer using gnosis safe")
+        .version("0.0.1");
+
+    program
+        .command("mint")
+        .description("mint ( ad-hoc mint )")
+        .argument("<string>", "destination( address )")
+        .argument("<string>", "amount( to transfer )")
+        .action(async (address: string, amount: string) => {
+            console.log("address", address);
+            console.log("amount", amount);
+            await main(address, amount);
+        });
+
+    program.parseAsync();
+}
+
+mint();
+
+async function initalizeSafe(existingAddress = EXISTING_SAFE_ADDRESS) {
+    console.log("initializeSafe safe address", EXISTING_SAFE_ADDRESS);
+    safeAddress = existingAddress;
+    const ethAdapterOwner1 = new EthersAdapter({
+        ethers,
+        signerOrProvider: owner1Signer,
+    });
+
+    safeSdkOwner1 = await Safe.create({
+        ethAdapter: ethAdapterOwner1,
+        safeAddress,
+    });
+}
+
+async function proposeMintTransaction(
+    amount: string,
+    to: string,
+    gasPrice: number
+) {
+    const WNCG_CONTRACT_ADDRESS = ethers.utils.getAddress(
+        process.env.WNCG_CONTRACT_ADDRESS!
+    );
+    const mintAmount = ethers.utils.parseUnits(amount, 18);
+
+    const contract = new ethers.Contract(
+        WNCG_CONTRACT_ADDRESS,
+        ["function mint(address account, uint256 amount) public"],
+        owner1Signer
+    );
+
+    const data = contract.interface.encodeFunctionData("mint", [
+        to,
+        mintAmount,
+    ]);
+
+    const safeTransactionData: SafeTransactionDataPartial = {
+        to: WNCG_CONTRACT_ADDRESS,
+        value: "0",
+        data,
+        gasPrice,
+    };
+
+    // Create a Safe transaction with the provided parametersx
```

Comment:
`parametersx` seems mis-typed. Can you update the comment?

Reply:
fixed!